lib/attrsets.nix

```diff
@@ -333,6 +333,66 @@ rec {
       ) (attrNames set)
     );
 
+   /*
+    Like builtins.foldl' but for Attrsets.
+    Iterates over every nameValuePair in the given AttrSet.
+    The result can be appended to `acc` which gets finally returned.
+
+    Attention:
+      There is a completely different function
+      'lib.foldAttrs'
```

Comment:
```suggestion
      `lib.foldAttrs`
```

This allows interpretation as markdown and can't be confused with `'` in identifiers (called "prime").